### ISSUE TITLE
Icons and Menu Very Small on Motorola Droid

### ISSUE THREAD

**Issue by GoogleCodeExporter** [google-code-export/appsorganizer#9]
```
On the Motorola Droid, when you select the shortcut on the home screen to
view the application folder, the icons and text is very small and difficult
to read.

I suspect this has to do with the increased resolution of the Droid.  The
apps are 6 rows across, and the spacing of the close graphic seems off as well.

I'd post a screenshot, but I'm not able to do that on a non-rooted phone.
```

Original issue reported on code.google.com by `ada...@gmail.com` on 12 Nov 2009 at 7:33

**Comment by GoogleCodeExporter** [comment]
```
Fixed in 1.32 Beta 2.
```

Original comment by `ada...@gmail.com` on 13 Nov 2009 at 8:47

**Comment by GoogleCodeExporter** [comment]
Original comment by `fabio.co...@gmail.com` on 16 Nov 2009 at 10:09
- Changed state: **Fixed**
